server/src/graql/parser/Parser.java

```diff
@@ -421,20 +423,13 @@ public Pattern visitPattern(GraqlParser.PatternContext ctx) {
     }
 
     @Override
-    public Conjunction<?> visitPattern_conjunction(GraqlParser.Pattern_conjunctionContext ctx) {
+    public Pattern visitPattern_conjunction(GraqlParser.Pattern_conjunctionContext ctx) {
```

Comment:
It's good practice to return the strict type whenever known. So this method should be returning `Conjunction<?>`.